app/Knight.py

```diff
@@ -0,0 +1,23 @@
+class Knight:
+    def __init__(self,
+                 name: str,
+                 power: int,
+                 hp: int,
+                 armour: int,
+                 weapon: int,
+                 potion: list,
```

Comment:
The `potion` parameter is annotated as a `list`, but it seems to be expected to be a `Potion` object based on the context in `app/Game.py`. Consider changing the type annotation to `Potion` for consistency.